lib/private/Server.php

```diff
@@ -198,6 +198,7 @@
 use OCP\Log\ILogFactory;
 use OCP\Mail\IMailer;
 use OCP\OCM\ICapabilityAwareOCMProvider;
+use OCP\OCM\IOCMProvider;
 use OCP\OCM\IOCMDiscoveryService;
```

Comment:
```suggestion
use OCP\OCM\IOCMDiscoveryService;
use OCP\OCM\IOCMProvider;
```

coding-standards complains